frontend/public/texts/project_texts.js

```diff
@@ -929,19 +917,15 @@ export default function getProjectTexts({ project, user, url_slug, locale, creat
       en: `Hey,
       I found this awesome climate project: "${project?.name}"${
         project?.creator &&
-        " created by " + (project.creator?.name ? project.creator?.name : creator?.name)
+        ` created by ` + (project.creator?.name ? project.creator?.name : creator?.name)
```

Comment:
```suggestion
        ` created by ${project.creator?.name ? project.creator?.name : creator?.name}`
```
The point of using the ` notation is that you don't need to build the string with "+".

Reply:
Same goes for other places where it's done like that